FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Models/Tasks/TaskStage.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Models.Tasks
+{
+    public enum TaskStage
```

Comment:
Is there a type for representing "task failed"? Or the failed tasks will be tracked by other means.

Reply:
This file is removed now.
When a task is failed with an exception thrown, `JobExecutor` will handle it.